### ISSUE TITLE
No HTTPS for openweathermap.org?

### ISSUE THREAD

**Issue by burdges** [jenslody/gnome-shell-extension-openweather#85]
It appears the extension using only HTTP not HTTPS for weather data coming from openweathermap.org https://github.com/jenslody/gnome-shell-extension-openweather/blob/master/src/openweathermap_org.js#L48 while other providers get HTTPS.  I confirmed that the API servers do not support HTTPS so I emailed them to suggest they add it.  It might help if others requested HTTPS support as well.

**Comment by jenslody** [comment]
This might be more easy (and less expensive) since https://letsencrypt.org/ is trusted and moves forward.

**Comment by burdges** [comment]
True.  Another approach is to convince someone to run an open weather map server on a hidden service accessible over tor as a .onion url.

**Comment by pizzamaker** [comment]
@burdges That would be the absolute ideal. Have you heard back from the devs?

**Comment by burdges** [comment]
Nope.

**Comment by rabidaudio** [comment]
This is somewhat of a big deal since the user's location is included in the request
